code/modules/clothing/under/miscellaneous.dm

```diff
@@ -275,6 +275,10 @@
 	item_state = "burial"
 	item_color = "burial"
 
+/obj/item/clothing/under/burial/Initialize()
+	. = ..()
+	verbs -= /obj/item/clothing/under/verb/toggle

```

Comment:
Doesn't it still have a random suit sensor var though?

Reply:
It inherits NO_SENSORS from /under